src/modules/editingTask.js

```diff
@@ -0,0 +1,44 @@
+import modifyTask from './modifyTask.js';
+
+function finishEditing(event) {
+  const selectedElementClass = `.${event.currentTarget.parentNode.parentNode.classList[0]}`;
+  const elementLi = document.querySelector(selectedElementClass);
+  const elementInput = event.currentTarget;
+  elementLi.classList.remove('editing-background');
+  elementInput.classList.remove('editing-background');
+  elementInput.addEventListener('input', modifyTask(elementInput.value, selectedElementClass));
+  const icon = document.querySelector(`${selectedElementClass} > img`);
+  icon.setAttribute('src', './assets/images/dot.png');
+}
+
+function editingTask(event) {
```

Comment:
- [ ] Please your app is not working. Try and work through then submit.
-
![sss](https://user-images.githubusercontent.com/63972670/167259552-b29dd783-e22e-4268-aa61-188cd8832462.JPG)

- [ ] Kindly stick to project requirements ([See Project](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md))

Reply:
@OLIPLICHE  Hello, I tested it and it worked, we needed to run it on a local server. to see it